src/panels/lovelace/entity-rows/hui-group-entity-row.ts

```diff
@@ -22,6 +23,21 @@ class HuiGroupEntityRow extends LitElement implements LovelaceRow {
 
   @internalProperty() private _config?: EntityConfig;
 
+  private _computeCanToggle = memoizeOne(
+    (hass: HomeAssistant, entityIds: string[]): boolean => {
```

Comment:
Just see you made a memoize from it, but you pass `hass`, that means it will still update on every state change.

Reply:
True...